src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -44,6 +44,16 @@ public TimeSpan AutomaticRefreshInterval
             }
         }
 
+        /// <summary>
+        /// 30 seconds is the time interval that must pass for <see cref="RequestRefresh"/> before obtain the first configuration.
+        /// </summary>
+        public TimeSpan BootstrapRefreshInterval { get; } = new TimeSpan(0, 0, 30);
```

Comment:
If we keep these public need @jmprieur to sign off. I think we can likely keep internal for first release and then add the ability to configure this in the future. If we make them settable publicly we should have a plan to quickly release a way of setting them via config.